probe/docker/container.go

```diff
@@ -389,20 +389,18 @@ func (c *container) getBaseNode() report.Node {
 	return result
 }
 
-func (c *container) controlsMap() map[string]report.NodeControlData {
+// Return a slice including all controls that should be shown on this container
+func (c *container) controls() []string {
 	paused := c.container.State.Paused
-	running := !paused && c.container.State.Running
-	stopped := !paused && !running
-	return map[string]report.NodeControlData{
-		UnpauseContainer: {Dead: !paused},
-		RestartContainer: {Dead: !running},
-		StopContainer:    {Dead: !running},
-		PauseContainer:   {Dead: !running},
-		AttachContainer:  {Dead: !running},
-		ExecContainer:    {Dead: !running},
-		StartContainer:   {Dead: !stopped},
-		RemoveContainer:  {Dead: !stopped},
+	switch {
+	case paused:
+		return []string{UnpauseContainer}
+	case c.container.State.Running:
+		return []string{RestartContainer, StopContainer, PauseContainer, AttachContainer, ExecContainer}
+	case !c.container.State.Running:
```

Comment:
Nit: why not like this?

```suggestion
	default:
```

If the reason is readability, I feel it's a bit odd to single out the Running condition from above as the real condition that applies here is `!paused && !running`.